guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin/index.md

```diff
@@ -114,6 +114,29 @@ You will need to use everything you know about string manipulation to get the la
 *   <a href='http://forum.freecodecamp.com/t/javascript-string-prototype-charat/15932' target='_blank' rel='nofollow'>JS String Prototype CharAt</a>
 *   <a href='http://forum.freecodecamp.com/t/javascript-string-prototype-concat/15935' target='_blank' rel='nofollow'>JS String Prototype Concat</a>
 
+## ![:sunflower:](https://forum.freecodecamp.com/images/emoji/emoji_one/sunflower.png?v=3 ":sunflower:") Intermediate Code Solution Alternative:
```

Comment:
Remove this heading and add `Second`

and under the upper heading `Intermediate Code Solution` and add `First` above the originally posted first code solution.

This will immensely increase readability and will also remove excessive headings.

Reply:
I have implemented your suggestions. 

If there is anything else I should change please tell me. This is my first time contributing to an open source project.